majilove/models.py

```diff
@@ -126,7 +129,16 @@ class Photo(MagiModel):
 
     release_date = models.DateField(_('Release date'), null=True, db_index=True)
     idol = models.ForeignKey(Idol, verbose_name=_('Idol'), related_name='photos', db_index=True)
+    '''
+    HOW_TO_OBTAIN_CHOICES = [
+        ('permanent', _('Permanent')), # N Ranked starter photos
+        ('photoshoot', _('Photoshoot')), #Gacha photos
+        ('event', _('Event')), #Event photos
+    ]
 
+    i_how_to_obtain = models.PositiveIntegerField(_('How to obtain'), choices=i_choices(HOW_TO_OBTAIN_CHOICES), db_index=True)
+    #@TODO expand this section to link photos to specific photo sets and events
+    '''
```

Comment:
WIP. Do we want a photoshoot vs event card distinction or just to link to a specific photo set / event?